api/all/docs/api_diff_1.1.0_vs_1.0.0.txt

```diff
@@ -0,0 +1,2 @@
+Comparing source compatibility of  against 
```

Comment:
Note: the lack of versions in the file appears to be due to this bug: https://github.com/melix/japicmp-gradle-plugin/issues/31

Reply:
I guess in the meantime it wouldn't be that hard for us to overwrite the file with a line that has an appropriate version if we want to. But no big deal

Reply:
definitely something we could enhance ourselves with a little sed or something.